apps/frontend/src/app/transformations-creator/transformations-creator.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+
+interface Transformation {
+  id: number;
+  name: string | null;
+  description: string | null;
+}
+
+@Component({
+  selector: 'app-transformations-creator',
+  templateUrl: './transformations-creator.component.html',
+  styleUrls: ['./transformations-creator.component.scss'],
+})
+
+export class TransformationsCreatorComponent {
+
+  transformations: Transformation[] = [];
+
+  get nameControl() {
+    return this.transformationForm.controls['name']
```

Comment:
This is one valid way to get the control of the form. 

I started to prefer the usage of the `get` function. This also supports passing an array to access nested controls. This might be handy for future use cases

[Documentation](https://angular.io/api/forms/AbstractControl#get)